porch/controllers/workloadidentitybinding/api/v1alpha1/workloadidentitybinding_types.go

```diff
@@ -31,7 +31,7 @@ type WorkloadIdentityBinding struct {
 	Status WorkloadIdentityBindingStatus `json:"status,omitempty"`
 }
 
-// WorkloadIdentityBindingSpec defines the desired state of RemoteRootSync
+// WorkloadIdentityBindingSpec defines the desired state of WorkloadIdentityBinding
 type WorkloadIdentityBindingSpec struct {
 	ServiceAccountRef ServiceAccountRef `json:"serviceAccountRef,omitempty"`
 	ResourceRef       ResourceRef       `json:"resourceRef,omitempty"`
```

Comment:
ResourceRef is an awkward name, if you have any better ideas :-)

Reply:
Updated the name.